docs/graphql/manual/auth/authentication/jwt.rst

```diff
@@ -103,6 +104,14 @@ mandatory, while the rest of them are optional.
    All ``x-hasura-*`` values should be of type ``String``, they will be converted to the
    right type automatically.
 
+.. note::
+
+   The JWT config should either have the ``claims_namespace`` or ``claims_namespace_path``
```

Comment:
This note should come in the below section. It feels too much to be in the example section.

Reply:
I'll update it